### PR TITLE
Specify security group for worker nodes, alb, vpe

### LINKED ISSUE #343: Surface ability to attach security groups to worker nodes, alb, vpe
### Description

https://cloud.ibm.com/docs/openshift?topic=openshift-vpc-security-group&interface=ui#vpc-sg-cluster

---

By submitting this issue, you agree to follow our [Code of Conduct](https://github.com/terraform-ibm-modules/documentation/blob/main/CODE_OF_CONDUCT.md)

### PR DESCRIPTION
**Pull request by vburckhardt**
### Description

fixes #343 

### Release required?

- [ ] No release
- [ ] Patch release (`x.x.X`)
- [x] Minor release (`x.X.x`)
- [ ] Major release (`X.x.x`)

### Run the pipeline

If the CI pipeline doesn't run when you create the PR, the PR requires a user with GitHub collaborators access to run the pipeline.

Run the CI pipeline when the PR is ready for review and you expect tests to pass. Add a comment to the PR with the following text:

```
/run pipeline
```

### Checklist for reviewers

- [ ] If relevant, a test for the change is included or updated with this PR.
- [ ] If relevant, documentation for the change is included or updated with this PR.

### For mergers

- Use a conventional commit message to set the release level. Follow the [guidelines](https://terraform-ibm-modules.github.io/documentation/#/merging.md).
- Include information that users need to know about the PR in the commit message. The commit message becomes part of the GitHub release notes.
- Use the **Squash and merge** option.

### CODE REVIEW

**Comment by vburckhardt** [comment]
do not merge until 1.62.0 is out. Currently on beta0.

File: version.tf
Comment (ocofaigh) [inline version.tf:7]: same change needed in `modules/fscloud/version.tf`

File: examples/multiple_mzr_clusters/version.tf
Comment (ocofaigh) [inline examples/multiple_mzr_clusters/version.tf:9]: This example is purposely using `>=` to test latest provider. There are other examples locked into lowest supported version

File: examples/fscloud/version.tf
Comment (ocofaigh) [inline examples/fscloud/version.tf:9]: This example is purposely using `>=` to test latest provider. There are other examples locked into lowest supported version

**Review by ocofaigh — CHANGES_REQUESTED** [review]
Hmm, you must be using an old common-dev-assets as I see you committed `module-metadata.json` but that hook was removed a while ago. Can you remove from this PR?

We will need a test to test the new example - do you want to add later? Also some updates needed if we want to onboard the example to catalog, but again can be done as follow up

File: examples/fscloud/version.tf
Comment (vburckhardt) [reply examples/fscloud/version.tf:9]: ahh it's wobbly, we probably want to review this as depends on tribal knowledge in the team.

Comment (ocofaigh) [reply examples/fscloud/version.tf:9]: its stated in the code comment above the logic

File: version.tf
Comment (vburckhardt) [reply version.tf:7]: fixed modules/fscloud/version.tf as well - it was missing < 2.0.0

File: examples/fscloud/version.tf
Comment (vburckhardt) [reply examples/fscloud/version.tf:9]: Yes, but the comment does not really resonate as it is relatively long, talk about 1 example, and is relatively far from the line that is changes.

Comment (vburckhardt) [reply examples/fscloud/version.tf:9]: I think this should be changed through the test stack- eg: have some dynamic change there on the version pinning so that there is fewer cognitive burden to understand / upgrade the example.

The core issue I think is that the examples as they are now are designed to be run in the test suite, which facilitate the job of the core team, but this is no really a matter for most consumers who are using those examples are a starting point / good example.

**Review by ocofaigh — APPROVED** [review]
LGTM - will create some follow up issues to track remaining items

**Pull request merged**
